controllers/notifications.js

```diff
@@ -0,0 +1,51 @@
+const notificationQuery = require('../models/notifications')
+/**
+ * Fetches the notifications data for current user
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+
+const getNotificationsForUser = async (req, res) => {
+  try {
+    let { page: currentPage, n: limit } = req.query
+    const { id: userId } = req.userData
+
+    if (!currentPage) {
+      currentPage = 1
+    }
+
+    if (!limit) {
+      limit = 10
+    }
+
```

Comment:
NIT:
you can pass default values for the variable while destructuring,
`const { page: currentPage=1 , n: limit=10 } = req.query` and avoid if statements